README.md

```diff
@@ -1 +1,91 @@
-# puppetlabs-aws_inventory
\ No newline at end of file
+# aws_inventory
+
+#### Table of Contents
+
+1. [Description](#description)
+2. [Requirements](#requirements)
+3. [Usage](#usage)
+
+## Description
+
+This module includes a Bolt plugin to generate Bolt targets from AWS EC2 instances.
+
+## Requirements
+
+You will need an `aws_access_key_id` and a `aws_secret_access_key` (see [providing aws credentials](https://docs.aws.amazon.com/amazonswf/latest/awsrbflowguide/set-up-creds.html)) in order to authenticate against aws API.
+
+## Usage
+
+The AWS Inventory plugin supports looking up running AWS EC2 instances. It supports several fields:
+
+-   `profile`: The [named profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) to use when loading from AWS `config` and `credentials` files. (optional, defaults to `default`)
+-   `region`: The region to look up EC2 instances from.
+-   `name`: The [EC2 instance attribute](https://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Instance.html) to use as the target name. (optional)
+-   `uri`: The [EC2 instance attribute](https://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Instance.html) to use as the target URI. (optional)
+-   `filters`: The [filter request parameters](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html) used to filter the EC2 instances by. Filters are name-values pairs, where the name is a request parameter and the values are an array of values to filter by. (optional)
+-   `config`: A Bolt config map where the value for each config setting is an EC2 instance attribute.
+
+One of `uri` or `name` is required. If only `uri` is set, then the value of `uri` will be used as the `name`.
+
+Accessing EC2 instances requires a region and valid credentials to be specified. The following locations are searched in order until a value is found:
+
+**Region**
+
+-   `region: <region>` in the inventory file
+-   `ENV['AWS_REGION']`
```

Comment:
It looks like these aren't actually supported anymore?

Reply:
the aws sdk library uses them in the background.